projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table.service.ts

```diff
@@ -88,4 +88,23 @@ export class ComparisonTableService {
   getComparisonTabs(tabIds: string[]): Observable<CMSComparisonTabComponent>[] {
     return tabIds.map(tabId => this.cmsService.getComponentData(tabId));
   }
+
+  /**
+   * Sorts and returns available tabs by a given property
+   *
+   * @param property property of the tab by which the tabs need to get sorted
+   * @returns CMSComparisonTabComponent[] observable
+   */
+  getAvailableTabsSortedByProperty(
+    property: string
+  ): Observable<CMSComparisonTabComponent[]> {
+    return this.availableTab$.pipe(
+      tap(tab => {
```

Comment:
Shouldn't this be `map` since you are transforming this observable? And not sure if we need it to be in the service since it's too specific and can be used only for Comparison table tabs.
Also, by leaving out curly braces the returns are implicit and don't need to be written.